back-end/totalRedeemedTotalUnredeemed.js

```diff
@@ -0,0 +1,85 @@
+const fs = require("fs"); //File system
+
+let unredeemed = [];
+let unused = [];
+var oneTotal = {};
+var oneRedeemed = {};
+var twoTotal = {};
+var twoRedeemed = {};
+var threeTotal = {};
+var threeRedeemed = {};
+function iterationObject(obj){
+    let inner_obj = {};    
+
+    for(prop in obj){
+        if(typeof(obj[prop])=="object"){
+            iterationObject(obj[prop]);
+        }
+        else{
+            if(prop=="Unredeemed Vouchers"){
+                inner_obj[prop] = obj[prop];
+                unredeemed.push(inner_obj);
+            }
+        }
+    }
+}
+function countKeysPerLevel(storage, level, obj, store) {
+  var keys = Object.keys(obj);
+  var count = keys.length;
+
+  storage[level] = (storage[level] || 0) + count;
+
+  for (var i = 0; i < count; i++) {
+    var childObj = obj[keys[store]];
+    if (typeof childObj === "object") {
+      countKeysPerLevel(storage, level + 1, childObj,store);
+    }
+  }
+}
+
+async function totalRedeemedTotalUnredeemed(req, res) {
+  let findRawPurchasesJson = fs.readFileSync("./backendData/rawpurchases.json");
+  let rawpurchasesJson = JSON.parse(findRawPurchasesJson);
+  let findRawRedemptionsJson = fs.readFileSync("./backendData/rawredemptions.json");
+  let rawredemptionsJson = JSON.parse(findRawRedemptionsJson);
+  let findtotalsJson = fs.readFileSync("./backendData/totals.json");
+  let totalsJson = JSON.parse(findtotalsJson);
+  var allStamps = Object.entries(totalsJson)
+  
+  var oneTotal = {};
+  var oneRedeemed = {};
+  var twoTotal = {};
+  var twoRedeemed = {};
+  var threeTotal = {};
+  var threeRedeemed = {};
+  countKeysPerLevel(oneTotal, 0, rawpurchasesJson, 0);
```

Comment:
Is this hard-coded for two locations and schemes? It should work for any number of locations and schemes